squad_client/commands/download_attachments.py

```diff
@@ -0,0 +1,86 @@
+from squad_client import logging
+from squad_client.core.command import SquadClientCommand
+from squad_client.core.models import Squad, TestRun
+from squad_client.shortcuts import get_build
+
+
+logger = logging.getLogger(__name__)
+
+
+class DownloadAttachmentsCommand(SquadClientCommand):
+    command = "download_attachments"
```

Comment:
Can this be `download-attachments` (`-` instead of `_`)? It's just so it goes with the "standard" on the other commands like `register-callback`, `submit-tuxsuite`...

Reply:
Done ✅